README.md

```diff
@@ -5,200 +5,168 @@
 
-## Schema
-~~~
-~~~
+```
+  AWS_ACCESS_KEY_ID=... AWS_SECRET_ACCESS_KEY=... docker-compose up
+```
 
-## Refile Data Model
+You should supply AWS creds from the `nypl-digital-dev` account so that the app can decrypt config from `./config/var_local.env`
 
-## ReCAP API RequestItem
-(https://uat-recap.htcinc.com:9093/swagger-ui.html#/)
+Navigate to http://local.nypl.org:8084/api/v0.1/recap/refile-requests
 
-## Requirements
+Note that use of docker-compose should be sufficient for testing local application code, but does not emulate running the app on a Node image as it does when deployed. This subtle environment difference should not matter unless something in the Node layer is under examination.
 
-* Node.js >=6.0
-* PHP >=7.0
-  * [pdo_pdgsql](http://php.net/manual/en/ref.pdo-pgsql.php)
+Note that because SIP2 connections are blocked from all but AWS infractructure, you can not perform a successful test of the app locally at writing. You can, however, perform a not successful test. To _attempt_ to perform a refile request, POST to `https://local.nypl.org:8084/api/v0.1/recap/refile-requests`:
+```
+{
+  "itemBarcode": "33433073352803"
+}
+```
+
+The process will hang for 30s and then mark the refile-request as failed (viewable at the GET endpoint noted above). You may use the `SKIP_SIP2` env var to slightly improve this testing experience.
 
-Homebrew is highly recommended for PHP:
-  * `brew install php71`
-  * `brew install php71-pdo-pgsql`
+To log into the running container:
+```
+docker exec -it refile-request-service_php_1 bash
+```
 
-## Installation
+To connect to the local db:
+```
+docker exec -it refile-service-postgres-db psql -U postgres refile_requests_local
+```
 
-1. Clone the repo.
-2. Install required dependencies.
-   * Run `npm install` to install Node.js packages.
-   * Run `composer install` to install PHP packages.
-   * If you have not already installed `node-lambda` as a global package, run `npm install -g node-lambda`.
-3. Setup [configuration](#configuration) files.
-   * Copy the `.env.sample` file to `.env`.
-   * Copy `config/var_qa.env.sample` to `config/var_qa.env` and `config/var_production.env.sample` to `config/var_production.env`.
+If you need to just build the app image described in the `Dockerfile`:
+```
+docker image build -t refile-request-service:local .
+```
 
-## Security
+### Running locally with SAM
 
-Authorization provided via OAuth2 authorization_code. Set scopes in the format of access_type:service.
-For example, read:holds to access the GET request method endpoints.
+To more fully emulate running the app in Node image:
 
-## Configuration
+Start local test postgres db:
 
-Various files are used to configure and deploy the Lambda.
+```
+docker-compose up -d db
+```
 
-### .env
+Edit `sam.local.yml` and change the `DB_CONNECT_STRING` to use your current public IP in place of "YOURIP". In OSX, your public IP is available via `ifconfig -u | grep 'inet ' | grep -v 127.0.0.1 | cut -d\  -f2 | head -1`.
```

Comment:
The `DB_CONNECT_STRING` in `sam.local.yml` does not seem to contain "YOURIP", can we clarify this?

Reply:
Good catch. Fixed